integration-tests/docker/environment-configs/middlemanager

```diff
@@ -19,6 +19,7 @@
 
 DRUID_SERVICE=middleManager
 DRUID_LOG_PATH=/shared/logs/middlemanager.log
+DRUID_MAIN=org.apache.druid.cli.Main
```

Comment:
I guess this technically isn't needed anymore, but maybe is still useful? Should we check if it isn't set and set this as a default value somewhere centralized?

Reply:
Thanks, I forgot to remove it. Since we can add custom services for integration tests, I'm not sure this variable could become useful for anything yet. I removed it for now. Maybe we can add it back later if needed.